ai/build.gradle

```diff
@@ -3,4 +3,7 @@ description = 'TripleA ai implementations'
 dependencies {
     implementation project(":game-core")
     implementation project(":java-extras")
+    implementation project(":swing-lib")
+    testImplementation project(":test-common")
+    testCompile project(':game-core').sourceSets.test.output
```

Comment:
Without this line, the ai tests don't work.  It needs the compiled test files from the game-core.

Reply:
I'm a bit concerned that we have a dependeny on test data from game-core. I'm interested to learn more about this.

Meanwhile, `testCompile` is legacy and will not be supported in a near future release of gradle. You should be able to swap this for `testImplementation`, does that cause issue doing so?